contracts/deploy/l2/01_deploy_L1Oracle.ts

```diff
@@ -5,7 +5,8 @@ import { deployUUPSProxiedContract } from "../utils";
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { sequencer, deployer } = await hre.getNamedAccounts();
-  await deployUUPSProxiedContract(hre, deployer, "L1Oracle", [sequencer]);
+  const bridger = "0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199"
+  await deployUUPSProxiedContract(hre, deployer, "L1Oracle", [bridger]);
```

Comment:
this is a temporary fix to avoid changing the sequencer nonce during deployment. will no longer be needed once sequencer updates the oracle
(hardcoded address instead of putting it in env due to the temp nature of this fix)